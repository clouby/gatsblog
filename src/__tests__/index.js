import React from "react";
import renderer from "react-test-renderer";
import { StaticQuery } from "gatsby";
import Index from "../pages/index";

beforeEach(() => {
  StaticQuery.mockImplementationOnce(({ render }) =>
    render({
      site: {
        siteMetadata: {
          title: "Macgyver",
        },
      },
    })
  );
});

describe("Index", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Index />).toJSON();

    console.log(tree);

    expect(tree).toMatchSnapshot();
  });
});
