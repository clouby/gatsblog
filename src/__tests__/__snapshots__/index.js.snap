// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
<div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "fontFamily": "Fira Code",
      }
    }
  >
    <h2
      style={
        Object {
          "marginBottom": 10,
        }
      }
    >
      Macgyver
    </h2>
    <p
      style={
        Object {
          "marginTop": 0,
          "opacity": 0.5,
        }
      }
    />
  </div>
</div>
`;
